source/board/pitaya_link.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file    pitaya_link.c
+ * @brief   board file for Pitaya-Link targeting nRF52 SoC
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2018, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_family.h"
+#include "target_board.h"
+
+extern target_cfg_t target_device_nrf52840_256;
+
+const board_info_t g_board_info = {
+    .info_version = kBoardInfoVersion,
+    .board_id = "1028",
+    .family_id = kNordic_Nrf52_FamilyID,
+    .daplink_url_name = "PRODINFOHTM",
+    .daplink_drive_name = "PITAYA-LINK",
+    .daplink_target_url = "https://wiki.makerdiary.com/pitaya-link/",
+    .target_cfg = &target_device_nrf52840_256,
+};
```

Comment:
I think it would be interesting to add the new CMSIS-DAP 2.1 attributes:
```
    .board_vendor = "xxxxx",
    .board_name = "xxxxx",
```

Reply:
Also in this file probably change `target_device_nrf52840_256` to `target_device_nrf52840`.

Reply:
Resolved as per comment